README.md

```diff
@@ -71,15 +72,17 @@ The goal of this project is to make the easiest, fastest, and most painless way
 
 ## Installation
 
+**Note: As Gitea is a [gogs](https://github.com/gogits/gogs) fork, tutorials and documentation related to gogs applies to Gitea too**
+
 Make sure you install the [prerequisites](https://gogs.io/docs/installation) first.
 
-There are 5 ways to install Gogs:
+There are 5 ways to install Gitea:
 
 - [Install from binary](https://gogs.io/docs/installation/install_from_binary.html)
 - [Install from source](https://gogs.io/docs/installation/install_from_source.html)
 - [Install from packages](https://gogs.io/docs/installation/install_from_packages.html)
```

Comment:
I would drop the gogs.io links, those would not be releases of Gitea anyway.
Maybe also drop the count of ways to install, as that number is going to change a few times before becoming stable :)